src/listeners/message.js

```diff
@@ -2,9 +2,25 @@ module.exports = {
     name: "message",
     exec: async (client, msg) => {
         if (!msg.guild) return;
-        if (msg.author.bot) return;
+        if (msg.author.bot) return;     
+       
+        const senpai = `<@!${client.user.id}>`
+		if (msg.content.toLowerCase().startsWith(senpai)) {			
```

Comment:
also, why don't merge this condition to line 22, it will look cleaner and shorter tho

Reply:
I tried it but the bot stopped working for me even with the prefix.  Yeah, and The bot will work even if there is No Nickname. That's Discord for you!